stdlib/int64.gr

```diff
@@ -281,18 +520,37 @@ export let rec gte = (x: Int64, y: Int64) => {
   ret
 }
 
-// Int64 bit/logical operations
+/**
+ * @section Bitwise Logic: Boolean operations on the bits of Int64 values.
+ */
+
+/**
+ * Computes the bitwise logical NOT of the given value.
```

Comment:
we could probably remove logical unless there is a bitwise not i dont know about and that is the correct symbol

Reply:
I'd stay away from giving an example of the operator since we don't have it in Grain (and it might imply that you can use that operator). And yes, I think it's good to drop the word logical.

Reply:
```suggestion
 * Computes the bitwise NOT of the given value.
```
Talked to Oscar about this one specifically and we don't want to show the operator for just this one because Grain doesn't have a bitwise NOT operator (yet).